src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 100;
+    private String randomColor = new ColorSupplier().getRandomColor();
```

Comment:
This is very bad practice. `ColorSupplier` should be a class-level variable and you should reuse it and call its method.